HabitTracker.harris-andy/README.md

```diff
@@ -0,0 +1,39 @@
+
+# Habit Logger Project
+
+This is my submission for the cSharpAcademy Habit Logger project found here: [Habit Logger Project](https://thecsharpacademy.com/project/12/habit-logger)
+
+
+## Project Description
+  - It's a small console CRUD app in which the user can track hobbies which are stored in a local SQLite database.
+  - Built with C#/.Net 8
+
+
+## Usage
+  - Follow the instructions and away you go
+  - i.e. Select from the menu to perform operations such as: viewing all records, inserting, updating and deleting records.
+
+    <img src="./images/gamemenu.png" alt="Game Menu" width="500"/>
```

Comment:
If you are just displaying an image you can do this in native markdown syntax:

`![Game Menu](./images/gamemenu.png)`

If the width being 500 is important to you, you can use an image editor to resize to 500px 😄 

This way you do not sully your markdown file with html.
Note: I break this rule myself as when I create my README's I like the app name and logo front and center.
[example](https://github.com/chrisjamiecarter/budget)